release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -244,6 +244,118 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
 existing CRD storage versions are not missing in the upgraded CRD, avoiding
 potential data loss.
 
+[id="ocp-4-5-operator-api"]
+==== Read-only Operator API (Technology Preview)
+
+The new Operator API is now available as a Technology Preview feature in
+read-only mode. Previously, installing Operators using Operator Lifecycle
+Manager (OLM) required cluster administrators to be aware of multiple APIs,
+including CatalogSources, Subscriptions, ClusterServiceVersions, and
+InstallPlans. This single Operator API resource is a first step towards a more
+simplified experience discovering and managing the lifecycle of Operators in a
+{product-title} cluster.
+
+Currently only available using the CLI and requiring a few manual steps to
+enable, this feature previews interacting with Operators as a first-class API
+object. Cluster administrators can discover previously installed Operators using
+this API in read-only mode.
+
+In a future release of {product-title}, cluster administrators will be able to
+use the Operator API to install an Operator bundle in a cluster without being
+part of a catalog, shortening the route for developers to test their packaging.
+
+To enable this Technology Preview feature:
+
+.Procedure
+
+. Disable xref:../architecture/architecture-installation.adoc#unmanaged-operators_architecture-installation[Cluster Version Operator (CVO) management] of the OLM:
++
+----
+$ oc patch clusterversion version \
+    --type=merge -p \
+    '{"spec": {"overrides":[{"kind": "Deployment", "name": "olm-operator", "namespace": "openshift-operator-lifecycle-manager", "unmanaged": true, "group": "apps"}]}}'
+----
+
+. Add the `OperatorLifecycleManagerV2=true` xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[FeatureGate]
+to the OLM Operator.
+
+.. Edit the OLM Operator's Deployment:
++
+----
+$ oc -n openshift-operator-lifecycle-manager \
+    edit deployment olm-operator
+----
+
+.. Add the following flag to the Deployment's `args` section:
++
+----
+...
+    spec:
+      containers:
+      - args:
+...
+        - --feature-gates
+        - OperatorLifecycleManagerV2=true
+----
+
+.. Save your changes.
+
+. Install an Operator using OLM if you have not already; for this example, an
+`etcd-test` Operator is installed in the project `test-project`.
+
+. Create the new Operator resource.
+
+.. Save the following to a file:
++
+.`etcd-test-op.yaml` file
+----
+apiVersion: operators.coreos.com/v2alpha1
+kind: Operator
+metadata:
+  name: etcd-test
+----
+
+.. Create the resource:
++
+----
+$ oc create -f etcd-test-op.yaml
+----
+
+. To have the Operator opt in to the new API, apply the
+`operators.coreos.com/etcd-test` label to the following objects related to your
+Operator:
++
+--
+* Subscription
+* InstallPlan
+* ClusterServiceVersion
+* Any CRDs owned by the Operator
```

Comment:
Owned and required by?

Reply:
I think it is Owned. Please @njhale correct me if I am wrong

Reply:
Automatic adoption in 4.6 will only add owned, so it's probably best to stick with that in the example (although it will pick up anything with the component label).